admin-dev/themes/new-theme/template/components/layout/search_form.tpl

```diff
@@ -33,7 +33,7 @@
     <a href="#" class="clear_search hide"><i class="icon-remove"></i></a>
   {/if}
   <div class="input-group">
-    <input type="text" class="form-control js-form-search" id="bo_query" name="bo_query" value="{$bo_query}" placeholder="{l s='Search (e.g.: product reference, customer name…)'} d='Admin.Navigation.Header'">
+    <input type="text" class="form-control js-form-search" id="bo_query" name="bo_query" value="{$bo_query}" placeholder="{l s='Search (e.g.: product reference, customer name…)' d='Admin.Navigation.Header'}" aria-label="{l s='Searchbar' d='Admin.Navigation.Header'}">
```

Comment:
I wonder how 'e.g.' will be read by screenreaders... Maybe it would be better to replace it with "example"?

Reply:
Screen reader shouldn't read the placeholder if it has an aria-label, it will read "Searchbar"

Reply:
Ok no problem then I guess